### ISSUE TITLE
Give admins ability to remove users from Connect chat channels

### ISSUE THREAD

**Issue by michael-tharrington** [forem/forem#9183]
**Is your feature request related to a problem? Please describe.**

Yes, occasionally users may need to be removed from Connect chat channels for various reasons and it'd be nice to have the ability to do so from https://dev.to/internal/chat_channels.

**Describe the solution you'd like**

From here - https://dev.to/internal/chat_channels, admins should have the ability to remove users from Connect chat channels.

I'm thinking we could change it so that the "Usernames to add" input is worded as "Usernames to add or remove". And, if the username is already in the chat channel, then it will be removed instead of added when an admin types in the username and clicks "Update Chat channel".

Also, if someone is removed from a channel in this way, it should be silent and avoid announcing who was removed from the channel.

**Additional Info**

Here's a screenshot of one of the chat channels in the list:

![Image 2020-07-07 at 11 11 05 AM](https://user-images.githubusercontent.com/16007075/86802009-977abf80-c042-11ea-9937-935c8df2ce35.png)

With the proposed solution, someone could type in "michaeltharrington" into the "Usernames to Add" input, click "Update Chat channel", and this would remove "michaeltharrington" from the channel.

**Comment by jacobherrington** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by Veggier** [comment]
I will try to work on it.

**Comment by nickytonline** [comment]
@Veggier, I've assigned this to you.

**Comment by michael-tharrington** [comment]
@Veggier appreciate the help! One thing I added to this feature request after the fact was _"Also, if someone is removed from a channel in this way, it should be silent and avoid announcing who was removed from the channel."_

Right now, a Connect channel admin can remove a user from a channel within Connect (not via https://dev.to/internal/chat_channels), but it doesn't do so silently. A message will appear that `@name` removed `@othername`.

I think if someone is removed from a channel via https://dev.to/internal/chat_channels this announcement shouldn't appear. Just let me know if you need more clarification. And thanks again!

**Comment by rhymes** [comment]
@Veggier after #9235 has been merged, is there anything else to be done here?
